go/vt/vtgate/tx_conn_test.go

```diff
@@ -132,7 +132,7 @@ func TestTxConnCommitOrderFailure1(t *testing.T) {
 	utils.MustMatch(t, &wantSession, session.Session, "Session")
 	assert.EqualValues(t, 1, sbc0.CommitCount.Get(), "sbc0.CommitCount")
 	// When the commit fails, we try to clean up by issuing a rollback
```

Comment:
We should add assertions about sbc1.CommitCount here, and sbc0.RollbackCount in the next test to make it clearer

Reply:
Added assertions though I'm not sure I understand them :)